_src/pldoc/static/js/ui.js

```diff
@@ -12,14 +12,13 @@ $(document).ready(function() {
 
     // smoothscroll to target links
     function smoothScrollLink(e) {
+        var this = e.target,
+            target = $(this).attr('href');
 
-        $.smoothScroll({
-            offset: -200,
-            easing: 'swing',
-            preventDefault: false,
-            speed: 1000,
-            scrollElement: null,
-            scrollTarget: $(this).attr('href')
+        $('html, body').stop().animate({
+            'scrollTop': $(target).offset().top
```

Comment:
Nit: It looks like the convention in this file is to not use quotes for field names.  Should this be `scrollTop: $(target).offset().top` instead?